app/src/main/java/com/akshay/newsapp/news/storage/NewsArticlesDao.kt

```diff
@@ -28,6 +28,12 @@ interface NewsArticlesDao {
         insertArticles(articles)
     }
 
+    /**
+     *  Get News article for given articleId
+     */
+    @Query("SELECT * FROM news_article WHERE id = :articleId")
```

Comment:
Minor suggestion to use constant
```suggestion
    @Query("SELECT * FROM ${NewsArticles.tableName} WHERE id = :articleId")
```